javalib/src/main/scala/java/time/Period.scala

```diff
@@ -0,0 +1,193 @@
+package java.time
+
+import scala.collection.JavaConverters._
+
+import java.time.chrono.{ChronoLocalDate, IsoChronology, ChronoPeriod}
+import java.time.temporal._
+
+final class Period private (years: Int, months: Int, days: Int)
+    extends ChronoPeriod with Serializable {
+  import ChronoUnit._
+
+  def get(unit: TemporalUnit): Long = unit match {
+    case YEARS  => years
+    case MONTHS => months
+    case DAYS   => days
+
+    case _ =>
+      throw new UnsupportedTemporalTypeException(s"Unsupported unit: $unit")
+  }
+
+  def getUnits(): java.util.List[TemporalUnit] =
```

Comment:
We have a format convention for `java.util`, we import it with `import java.{util => ju}` and use it like `ju.List[TemporalUnit]`.